core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -496,6 +496,45 @@ public int get(PageId pageId, int pageOffset, int bytesToRead, PageReadTargetBuf
     }
   }
 
+  /**
+   * delete the specified page.
+   * @param pageId page identifier
+   * @param cacheContext cache context
+   * @return whether the page is deleted successfully or not
+   */
+  public boolean delete(PageId pageId, CacheContext cacheContext) {
```

Comment:
This method seems to repeat quite a lot of code with the next method. Would it be possible to merge or write a help function for saving the repeating?